site2/docs/admin-api-brokers.md

```diff
@@ -48,6 +48,33 @@ admin.brokers().getActiveBrokers(clusterName)
 
 <!--END_DOCUSAURUS_CODE_TABS-->
 
+### Get the information of the leader broker
+
+Fetch the information of the leader broker.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--pulsar-admin-->
+
+```shell
+$ pulsar-admin brokers leader-broker
+```
+
+```
+broker1.use.org.com:8080
+```
+
+<!--REST API-->
+
+{@inject: endpoint|GET|/admin/v2/brokers/leaderBroker?version=[[pulsar:version_number]]}
+
+<!--JAVA-->
+
+```java
+admin.brokers().getLeaderBroker()
+```
+
```

Comment:
Same comment as above.

Reply:
The URL was contained in this doc.
The brokers method of the {@inject: javadoc:PulsarAdmin:/admin/org/apache/pulsar/client/admin/PulsarAdmin.html} object in the Java API

Reply:
Thanks for your explanations. Sorry I do not get you. I mean we can add a link w/ the line pointed to this command? Like https://github.com/apache/pulsar/blob/master/pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/BrokersImpl.java#L80

Reply:
ok